docker/arm.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM arm64v8/alpine:3.16.2
```

Comment:
This should be bumped to the same version of alpine that's in [docker/Dockerfile](https://github.com/AndrewPaglusch/FlashPaper/blob/master/docker/Dockerfile).